### PR TITLE
[Security] add CAS 2.0 AccessToken handler

### LINKED ISSUE #19490: [Security] add CAS 2.0 AccessToken handler
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#48276
| PR author(s) | @nacorp
| Merged in    | 7.1

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 7.1 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alamirault**
Try fix #19490

### CODE REVIEW

File: security/access_token.rst [hunk 1]
```diff
@@ -697,6 +697,187 @@ create your own User from the claims, you must
         }
     }
 
+Using CAS 2.0
+-------------
+
+`Central Authentication Service (CAS)`_ is an enterprise multilingual single
+sign-on solution and identity provider for the web and attempts to be a
+comprehensive platform for your authentication and authorization needs.
```
Comment (alamirault) [inline security/access_token.rst:705]: I never used CAS, we can probably have a better description

@nacorp your suggestions are welcomed

Comment (nacorp) [reply security/access_token.rst:705]: Ow you've been so quick.  #19490  was on top of my todo. I'll come by to check what you wrote and complete if necessary as soon as I can. Thanks a lot for your help :-)

Comment (javiereguiluz) [reply security/access_token.rst:705]: @nacorp friendly ping about this in case you have some time to review it. Thanks a lot!

Comment (nacorp) [reply security/access_token.rst:705]: I'm on it. The description provided by alamirault is ok. I want to test the documentation he wrote on a real project but I miss some time at the moment.

Comment (nacorp) [reply security/access_token.rst:705]: I'm so sorry for being that late - it sound great for me ! (thanks @alamirault  !)

**Comment by javiereguiluz** [comment]
Antoine, I'm glad to tell you that this was finally merged. Thanks a lot for your nice work here 🙌

**Pull request merged**